packages/shared/src/permissions/permissions.utils.ts

```diff
@@ -2,8 +2,266 @@ import {
   Permission,
   UserPermissions,
   MemberRole,
+  PermissionsObject,
+  OrganizationInterface,
+  DefaultMemberRole,
+  Role,
 } from "back-end/types/organization";
 
+export const POLICIES = [
+  "ReadData",
+  "Comments",
+  "FeaturesFullAccess",
+  "ArchetypesFullAccess",
+  "FeaturesBypassApprovals",
+  "ExperimentsFullAccess",
+  "VisualEditorFullAccess",
+  "SuperDeleteReports",
+  "DataSourcesFullAccess",
+  "DataSourceConfiguration",
+  "RunQueries",
+  "MetricsFullAccess",
+  "FactTablesFullAccess",
+  "FactMetricsFiltersFullAccess",
+  "DimensionsFullAccess",
+  "SegmentsFullAccess",
+  "IdeasFullAccess",
+  "PresentationsFullAccess",
+  "SDKPayloadPublish",
+  "SDKConnectionsFullAccess",
+  "AttributesFullAccess",
+  "EnvironmentsFullAccess",
+  "NamespacesFullAccess",
+  "SavedGroupsFullAccess",
+  "GeneralSettingsFullAccess",
+  "NorthStarMetricFullAccess",
+  "TeamManagementFullAccess",
+  "ProjectsFullAccess",
+  "TagsFullAccess",
+  "APIKeysFullAccess",
+  "IntegrationsFullAccess",
+  "EventWebhooksFullAccess",
+  "BillingFullAccess",
+  "AuditLogsFullAccess",
+] as const;
+
+export type Policy = typeof POLICIES[number];
+
+export const POLICY_PERMISSION_MAP: Record<Policy, Permission[]> = {
+  ReadData: ["readData"],
+  Comments: ["readData", "addComments"],
+  FeaturesFullAccess: [
+    "readData",
+    "manageFeatureDrafts",
+    "manageFeatures",
+    "canReview",
+  ],
+  ArchetypesFullAccess: ["readData", "manageArchetype"],
+  FeaturesBypassApprovals: [
+    "readData",
+    "manageFeatureDrafts",
+    "manageFeatures",
+    "canReview",
+    "bypassApprovalChecks",
+  ],
+  ExperimentsFullAccess: ["readData", "createAnalyses"],
+  VisualEditorFullAccess: ["readData", "manageVisualChanges"],
+  SuperDeleteReports: ["readData", "superDeleteReport"],
+  DataSourcesFullAccess: [
+    "readData",
+    "createDatasources",
+    "editDatasourceSettings",
+  ],
+  DataSourceConfiguration: ["readData", "editDatasourceSettings"],
+  RunQueries: ["readData", "runQueries"],
+  MetricsFullAccess: ["readData", "createMetrics"],
+  // TODO: Fix fact permissions
+  FactTablesFullAccess: ["readData", "manageFactTables", "createMetrics"],
+  FactMetricsFiltersFullAccess: ["readData", "createMetrics"],
+  DimensionsFullAccess: ["readData", "createDimensions"],
+  SegmentsFullAccess: ["readData", "createSegments"],
+  IdeasFullAccess: ["readData", "createIdeas"],
+  PresentationsFullAccess: ["readData", "createPresentations"],
+  SDKPayloadPublish: ["readData", "publishFeatures", "runExperiments"],
+  // TODO: add permissions
+  SDKConnectionsFullAccess: ["readData"],
```

Comment:
Seems like we'd want to add the following env_scoped permissions: `createSDKConnections`, `updateSDKConnections`, and `deleteSDKConnections`.

Reply:
We have also introduced methods to split `SDKWebhooks` and `EventWebhooks`. 

What do you think about adding a new permission `manageSDKWebhooks` and then adding that to the `SDKConnectionsFullAccess` array?

If we do it, we can then always update the existing `manageWebhooks` to be `manageEventWebooks` to be more contextual

Reply:
The only complexity with breaking `manageWebhooks` into `manageSDKWebhooks` and `manageEventWebhooks` is that currently `manageWebhooks` is a `GLOBAL_PERMISSION` currently only given to admins. It really feels like `manageSDKWebhooks` should be a `PROJECT_SCOPED_PERMISSION`.

We could make build it in a way that wouldn't affect existing users - we'd just make sure that only admins get the permission, but we would need to refactor the `can{action}SDKWebhook` methods.

Reply:
I've got this work queued up in a PR [here](https://github.com/growthbook/growthbook/pull/2503).